<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSPoreStructureAnalysis</class>
 <widget class="QMainWindow" name="RSPoreStructureAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>岩样孔隙结构分析模块</string>
  </property>
  <widget class="QWidget" name="layout_centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="layout_Well">
      <item>
       <widget class="QLabel" name="wWellLabel">
        <property name="text">
         <string>井名:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="wWellEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="wWellSelectBtn">
        <property name="text">
         <string>选井...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="wTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="wSanningTab">
       <attribute name="title">
        <string>扫描岩样数据</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>输出信息：</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QTextEdit" name="wReadInfoEdit">
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>参数：</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="minimumSize">
                    <size>
                     <width>84</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>放大倍数：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="wKlpEdit">
                   <property name="text">
                    <string>1500</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>修正放大倍数：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="wRrfEdit">
                   <property name="text">
                    <string>0.468</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>像素单元尺度：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="wDminEdit">
                   <property name="text">
                    <string>1</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="minimumSize">
                    <size>
                     <width>84</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>孔隙度：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="wPorEdit">
                   <property name="text">
                    <string>0.2</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>文字比例：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="wDrmEdit">
                   <property name="text">
                    <string>0.13</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="wPictureBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>104</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>扫描图片</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="wTerminateBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>104</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>中止计算</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wDataBaseTab">
       <attribute name="title">
        <string>数据库</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="4">
         <widget class="QTableView" name="wTableView"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="wImportDataBtn">
          <property name="text">
           <string>导入数据...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="wSaveBtn">
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="wCrossPlotBtn">
          <property name="text">
           <string>交汇显示...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="wExitBtn">
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>wWellSelectBtn</sender>
   <signal>clicked()</signal>
   <receiver>RSPoreStructureAnalysis</receiver>
   <slot>slot_WellSelectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wPictureBtn</sender>
   <signal>clicked()</signal>
   <receiver>RSPoreStructureAnalysis</receiver>
   <slot>slot_PictureBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wImportDataBtn</sender>
   <signal>clicked()</signal>
   <receiver>RSPoreStructureAnalysis</receiver>
   <slot>slot_ImportDataClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wSaveBtn</sender>
   <signal>clicked()</signal>
   <receiver>RSPoreStructureAnalysis</receiver>
   <slot>slot_SaveBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wCrossPlotBtn</sender>
   <signal>clicked()</signal>
   <receiver>RSPoreStructureAnalysis</receiver>
   <slot>slot_CrossPlotClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wTerminateBtn</sender>
   <signal>clicked()</signal>
   <receiver>RSPoreStructureAnalysis</receiver>
   <slot>slot_TerminateClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wExitBtn</sender>
   <signal>clicked()</signal>
   <receiver>RSPoreStructureAnalysis</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_WellSelectClicked()</slot>
  <slot>slot_PictureBtnClicked()</slot>
  <slot>slot_ImportBtnClicked()</slot>
  <slot>slot_ImportDataClicked()</slot>
  <slot>slot_SaveBtnClicked()</slot>
  <slot>slot_CrossPlotClicked()</slot>
  <slot>slot_TerminateClicked()</slot>
 </slots>
</ui>
